__tests__/__snapshots__/ord.test.js.snap

```diff
@@ -100,6 +100,64 @@ exports[`Tests for default ORD document Successfully create ORD Documents with d
     },
   ],
   "description": "this is an application description",
+  "eventResources": [
+    {
+      "description": "This is an example event catalog that contains only a partial ODM cap-js-ord V1 event",
+      "extensible": {
+        "supported": "no",
+      },
+      "ordId": "capjs.ord:eventResource:undefined.AdminService:v1",
```

Comment:
I still feel bad about all those `undefined` here. But this is planned to be fixed with a follow up ADR?

Reply:
Yeah, I will fix it in next PR

Reply:
The `undefined` issues are all related and coming from the `global.capNamespace`. That will also fix this.